src/components/DayPicker.jsx

```diff
@@ -447,37 +461,49 @@ export default class DayPicker extends React.Component {
     return (
       <div className={dayPickerClassNames} style={dayPickerStyle} >
         <OutsideClickHandler onOutsideClick={onOutsideClick}>
-          {!verticalScrollable && this.renderNavigation()}
-
-          <div className="DayPicker__week-headers">
+          <div
+            className="DayPicker__week-headers"
+            aria-hidden="true"
+            role="presentation"
+          >
             {weekHeaders}
           </div>
 
           <div
-            className={transitionContainerClasses}
-            ref={(ref) => { this.transitionContainer = ref; }}
-            style={transitionContainerStyle}
+            className="DayPicker__focus-region"
+            ref={(ref) => { this.container = ref; }}
+            role="region"
+            tabIndex={0}
```

Comment:
is this something you specifically want users to be able to tab to or just to focus? Seems like something that should be more for focusing purposes, in which case you'd want a tabindex of -1